.internal/aws/cloudformation/macro.yaml

```diff
@@ -38,6 +38,23 @@ Resources:
                         }
                     }
 
+            if "ElasticServerlessForwarderKinesisEvents2" in parameters:
```

Comment:
Overall it looks good to me. Not sure if it's worth extracting all the sub functions like this:
```
def process_sqs_events(events, events_fragment):
    for sqs_event in events:
        sqs_event = sqs_event.strip()
        if len(sqs_event) == 0:
            continue
        sqs_event_name = f"SQSEvent{hex_suffix(sqs_event)}"
        events_fragment[sqs_event_name] = {
            "Type": "SQS",
            "Properties": {
                "Queue": sqs_event,
                "BatchSize": 10,
                "Enabled": True,
            }
        }

if "ElasticServerlessForwarderSQSEvents" in parameters:
    process_sqs_events(parameters["ElasticServerlessForwarderSQSEvents"], events_fragment)

if "ElasticServerlessForwarderSQSEvents2" in parameters:
    process_sqs_events(parameters["ElasticServerlessForwarderSQSEvents2"], events_fragment)
```
Not a required change, either way works!